pkgs/applications/editors/neovim/neovide/default.nix

```diff
@@ -20,7 +20,18 @@
 , stdenv
 , enableWayland ? stdenv.isLinux
 , wayland
+, xcbuild
 , xorg
+# Apple frameworks
```

Comment:
I think it is more common to pass the package darwin.

Reply:
yes, but it is dirty and shouldn't be done.